cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/analyzer/ExcludedBrokersForLeadershipTest.java

```diff
@@ -265,7 +301,8 @@ public ExcludedBrokersForLeadershipTest(int testId,
                                           Boolean expectedToOptimize) {
     _testId = testId;
     _goal = goal;
-    _optimizationOptions = new OptimizationOptions(Collections.emptySet(), excludedBrokersForLeadership);
+    _optimizationOptions = new OptimizationOptions(Collections.emptySet(), excludedBrokersForLeadership,
+                                                   Collections.emptySet(), false);
```

Comment:
What is the purpose of this change?
Looks like previous code was already doing the same thing with fewer arguments. Can we rollback this change?

Reply:
Yes, rolled back